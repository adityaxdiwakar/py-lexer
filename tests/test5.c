void someFunction() {
    int x;
    if (x > 0) {
        int y = 10;
    } else {
       int y = -10;
    }
}
