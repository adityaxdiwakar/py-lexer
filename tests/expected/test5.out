VOID_WORD
IDENT(someFunction)
LPAREN
RPAREN
LBRACK
INT_WORD
IDENT(x)
SEMI
IF
LPAREN
IDENT(x)
GT
INT(0)
RPAREN
LBRACK
INT_WORD
IDENT(y)
EQUALS
INT(10)
SEMI
RBRACK
ELSE
LBRACK
IDENT(y)
EQUALS
SUB
INT(10)
SEMI
RBRACK
RBRACK
